src/darwin/Framework/CHIP/MTRAttestationTrustStoreBridge.mm

```diff
@@ -41,3 +41,27 @@
     }
     return CHIP_ERROR_CA_CERT_NOT_FOUND;
 }
+
+CHIP_ERROR MTRAttestationTrustStoreBridge::GetCertificationDeclarationSigningKey(
+    const chip::ByteSpan & skid, chip::Crypto::P256PublicKey & pubKey) const
+{
+    VerifyOrReturnError(skid.size() == chip::Crypto::kSubjectKeyIdentifierLength, CHIP_ERROR_INVALID_ARGUMENT);
+
+    size_t cdIdx;
+    chip::ByteSpan candidate;
+
+    for (cdIdx = 0; cdIdx < mCDCerts.count; ++cdIdx) {
+        uint8_t skidBuf[chip::Crypto::kSubjectKeyIdentifierLength] = { 0 };
+        candidate = asByteSpan(mCDCerts[cdIdx]);
```

Comment:
Should use the shared`AsByteSpan` helper instead of the duplicate version in this file... Followup ok for this part.